fastembed/text/onnx_embedding.py

```diff
@@ -72,19 +72,27 @@
         "size_in_GB": 0.13,
         "sources": {
             "url": "https://storage.googleapis.com/qdrant-fastembed/fast-bge-small-en-v1.5.tar.gz",
+        },
+    },
+    {
+        "model": "BAAI/bge-small-en-v1.5-quantized",
+        "dim": 384,
+        "description": "Fast English model",
+        "size_in_GB": 0.06,
+        "sources": {
```

Comment:
https://github.com/qdrant/fastembed/blob/main/docs%2Fexamples%2FSupported_Models.ipynb should be rerun to include this.

Reply:
I'll do it, thank you :)